test/IntelFPGAFunctionAttributes.ll

```diff
@@ -1,7 +1,8 @@
 ;; Can be compiled using https://github.com/intel/llvm SYCL compiler from:
 ;; class Foo {
 ;; public:
-;;   [[intelfpga::max_global_work_dim(1),
+;;   [[intelfpga::uses_global_work_offset(0),
```

Comment:
Can you also please test that uses_global_work_offset(0) **does not** set the Execution Mode/metadata?